packages/renderer/src/lib/kube/KubeActions.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+import KubeApplyYamlButton from '../kube/KubeApplyYAMLButton.svelte';
```

Comment:
here you could use the `/@/` import scheme

Reply:
Fixed :+1: